pkg/client/logs_test.go

```diff
@@ -0,0 +1,185 @@
+package client
```

Comment:
are there any tests with the "headers?"

Reply:
all the files should have the headers, i'll add it here and submit another PR for any other missing headers